### ISSUE TITLE
feature: Share design/file with others who are not on your team

### ISSUE THREAD

**Issue by jdittrich** [uxbox/uxbox#2739]
### Is your feature request related to a problem? Please describe.

Two scenarios here: 

1) I coach some people in design occasionally, so I want to work with them for some sessions and then I might not work with them for some time. What I imagined was asking the mentees to create a penpot account and then invite them to collaborate on a file; even better it would have been if I could invite people without account.
2) I work on an open source project. There is a core team, but often there are people who might become core design contributors over time or people who take an specific interest e.g. if technology and design are intertwined and a dev wants to get design involved for this. 

### Describe the solution you'd like.

* In the main design view, have a sharing button/link
* Have it open a dialog for managing access of accounts (accounts outside the team; team always has access if file is stored at the team); similar to what already can be done for sharing prototypes (which is also hard to discover, #855)

I guess google suite has the "standard" UI for that:

<img width="368" height="272" alt='image showing an invite collaborators dialog. There is a list on accounts that can "edit" or "view" as well an a general option for editing merely via link (or not) as well as a note saying that all team members have access to the file' src="https://github.com/user-attachments/assets/3a5e284a-371a-4b3b-ab78-9fdd49d4f761" />

### Describe alternatives you've considered.

UI
* It could also be an option in the "file" menu (or you could have it in both places) 

Feature itself:
* You could leave it as it is and conciously position penpot as focussing on relatively static, project-internal teams.

**Comment by mollyclare** [comment]
The fact that all access controls are at the project level, not the document level, makes it really hard for looser communities to use Penpot. 
**My use case:**
My colleagues and I will work with various FOSS/civic tech teams for between 30 minutes and 10 hours giving them design support, but then we probably will not work with them again, at least not for a couple years. With the current model, we would have to make a new project containing 1 file for each each time we offer design support. This isn't practical for several reasons:
- getting them set up with an account that they won't really use will take 10-15 minutes of our time
- I will have accumulated 20+ projects within a couple months
- I can't clean up old projects, because then the folks I worked with lose access to our work

**Proposed solution:**
- Set access controls for who can edit each file in the upper right, where the "play" button currently is. Use a "share" icon. "Team members only," "anyone with the link," and "private" are the settings I need. Being able to invite someone by email would be a cherry on top.

**Comment by catcargos** [comment]
Permissions on a file level is have to be.

**Comment by thisbit** [comment]
Hi, just want to add my bit on this feature request.
I work with teams sometimes from designer side, with devs and clients on the other, and sometimes as a dev with designers and clients on the other. (not both sides on same projects, sure)

I also teach design to groups around 15 over long periods (1 semester).

So in cases of team work, as a teacher or as a collaborator Penpot shines with team management.
But for sharing with clients I really need to be able to share the design view rather then the prototype view.

Interactive prototypes are less commonly used, mostly just to clarify some features.

So what would be needed would be to share the link to the file with control to "just view" for anyone with link, to comment for anyone with a certain authentication, for instance email. Or some other form of auth.

Hope this is planned to be included in future update ...
Thanks

**Comment by steveoriol** [comment]
Yes, I think as a first step, what can be done quite simply is to add "anonymous" or "everyone" to this list, then make it so that for that choice, the share link can see the "info" and "code" in the "Inspect (G I)" tab without needing an account.
As this screenshot:
![Screenshot of the list to modify](https://www.netxtime.com/sites/default/files/2023-08/Sélection_015.png)

**Comment by madalenapmelo-kp** [comment]
Thank you all for sharing your use cases! 

We have already a few User Stories in our backlog to improve the permission level, I have linked this issue there so that these use cases will be considered as we better define this feature. You can find the details here: https://tree.taiga.io/project/penpot/us/6590

**Comment by schelmo** [comment]
Using penpot to create designs, it would be really helpfull to allow anonymous users to at least inspect via the shared link.
This is for example possible in adobe xd (there you can even create comments anonymously).

This is currently even possible when changing the shared url from:
`https://<penpothost>/#/view/<uid>?page-id=<pageId>&section=interactions&frame-id=<frameId>&index=0&share-id=<shareId>`
to
`https://<penpothost>/#/view/<uid>?page-id=<pageId>&section=inspect&frame-id=<frameId>&index=0&share-id=<shareId>`
(changing the the `section=interactions` to `section=inspect`)
There is no login prompt when changing the url and you can inspect everything.
(but opening the inspect link in a new tab doesnt work somehow).
So i think it should be trivial to disable the login prompt (maybe defined by some setting like @steveoriol suggested)

Creating new users just for let them inspect a design is a little bit overkill for me (I selfhost penpot and I don't want the developers to create projects/teams and so on).

Just sharing the link to allow inspection would be really nice!

**Comment by HeikoTietze** [comment]
My use case: I work on UX/UI topics for LibreOffice, and part of the work is to ponder with volunteers about redesign proposals. For example https://bugs.documentfoundation.org/show_bug.cgi?id=156598 attachment "mockup".

The ideal workflow would be to have the draft with full r/w access for everyone, without the need to register and invitation. Once the draft is done I want to make it r/o but public accessible. For example I ask on social media for thumbs up/down. And in some case it could be useful to limit access to the owner or registered contributors, aka the team. I surely want to limit management function like create or delete to people with raised access. And who can do that should have an even higher permission.

Essentially something like file permissions and attributes on Linux.

**Comment by jcklpe** [comment]
Seconding this request.

**Comment by marioandrearaujo** [comment]
My use case is simple: I just want to share a design file with the world, view-only, no account required. Today the only way is the view-mode "Share prototype" link, which does not offer a similar experience as the main editor does.
